src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
```

Comment:
The import statement for 'java.lang.reflect.Array' is not used in this class and should be removed to keep the code clean.